src/pages/ProfilePage/index.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  DONE_BUTTON_TEXT,
+  EDIT_BUTTON_TEXT,
+  PLACEHOLDER_DEFAULTS,
+} from '@/constants/profile';
+import { useCheckAuthUser } from '@/hooks/useAuth';
+import { useChangeImage } from '@/hooks/useGetProfile';
+
+import {
+  ImageWrapper,
+  Label,
+  ProfileBackGroundImage,
+  ProfileTopWrapper,
+  ProfileWrapper,
+  UserInfoWrapper,
+  UserWrapper,
+} from './style';
+
+export default function ProfilePage() {
+  const [isEditing, setIsEditing] = useState<boolean>(false);
+  const [introduction, setIntroduction] = useState<string>('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const { changeImage } = useChangeImage(setIsLoading);
+  const { data: authUser } = useCheckAuthUser();
+  const params = useParams();
+  const userId = params.userId;
+
+  const handleFileChange = (file: File | null) => {
+    if (file) {
+      changeImage(file);
+    }
+  };
+
+  const handleEditButtonClick = () => {
+    setIsEditing((prevIsEditing) => !prevIsEditing);
+  };
+
+  const handleFormSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setIsEditing((prevIsEditing) => !prevIsEditing);
+  };
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.value.length < 17) {
+      setIntroduction(e.target.value);
+    } else {
+      alert('작성 범위를 초과했습니다.');
+    }
+  };
+
+  const placeholderText =
+    introduction === '' ? PLACEHOLDER_DEFAULTS : introduction;
+
+  const buttonText = isEditing ? DONE_BUTTON_TEXT : EDIT_BUTTON_TEXT;
+
+  const defaultImage = '../../public/vite.svg';
+  return (
+    <>
+      {authUser && (
+        <>
+          {authUser._id === userId ? (
+            <ProfileWrapper>
+              <ProfileTopWrapper>
+                <ProfileBackGroundImage>
+                  <UserInfoWrapper>
+                    {isLoading ? (
+                      <Skeleton
+                        style={{ marginTop: '1.2rem' }}
+                        width="80px"
+                        height="80px"
+                        borderRadius="50%"></Skeleton>
+                    ) : (
+                      <ImageWrapper>
+                        <ImageUpload
+                          onFileChange={handleFileChange}
+                          ratio="5/5"
+                          width="80px"
+                          borderRadius="50%"
+                          image={
+                            authUser?.image
+                              ? `${authUser.image}?${Date.now()}`
+                              : defaultImage // 초기 값
+                          }
+                        />
+                      </ImageWrapper>
+                    )}
+                    <UserInfo
+                      userName={authUser?.fullName}
+                      userId={authUser?.email}
+                    />
+                  </UserInfoWrapper>
+                </ProfileBackGroundImage>
```

Comment:
```suggestion
                <ProfileBackGroundImage>내정보</ProfileBackGroundImage>
                <UserInfoWrapper>
                  {isLoading ? (
                    <Skeleton
                      style={{ marginTop: '1.2rem' }}
                      width="80px"
                      height="80px"
                      borderRadius="50%"></Skeleton>
                  ) : (
                    <ImageWrapper>
                      <ImageUpload
                        onFileChange={handleFileChange}
                        ratio="5/5"
                        width="80px"
                        borderRadius="50%"
                        image={
                          authUser?.image
                            ? `${authUser.image}?${Date.now()}`
                            : defaultImage // 초기 값
                        }
                      />
                    </ImageWrapper>
                  )}
                  <UserInfo
                    userName={authUser?.fullName}
                    userId={authUser?.email}
                  />
                </UserInfoWrapper>
```
`UserInfoWrapper` 위에 backgroundimage를 배치해봤습니다